package.json

```diff
@@ -6,17 +6,20 @@
   "bugs": {
     "url": "https://github.com/blinkmobile/angular-location.js/issues"
   },
-  "dependencies": {},
+  "dependencies": {
+    "@blinkmobile/geolocation": "^2.0.0",
```

Comment:
yeah I think a standard about this would be useful, i'm always wondering if i should allow auto patch updates (seems like a good idea) but after kynans experience with npm package update breaking the program leading to a hard to track down bug im worried about that happening.

not a big deal here as we control the geolocation one, and not a deal breaker for me